src/config/mekanism.cfg

```diff
@@ -1,244 +1,699 @@
+
+    # Log Mekanism packet names. Debug setting.
     B:LogPackets=false
+
+    # Maximum Joules per mB of Steam. Also affects Thermoelectric Boiler.
     D:MaxEnergyPerSteam=100.0
+
+    # Flamethrower Gas Tank capacity in mB.
     I:MaxFlamethrowerGas=24000
+
+    # Jetpack Gas Tank capacity in mB.
     I:MaxJetpackGas=24000
+
+    # Fluidic Plenisher stops after this many blocks.
     I:MaxPlenisherNodes=4000
+
+    # Maximum block distance to pull fluid from for the Electric Pump.
     I:MaxPumpRange=80
+
+    # Scuba Tank Gas Tank capacity in mB.
     I:MaxScubaGas=24000
+
+    # Energy multiplier for using silk touch mode with the Digital Miner.
     I:MinerSilkMultiplier=6
+
+    # Radius of the explosion of Obsidian TNT.
     I:ObsidianTNTBlastRadius=12
+
+    # Fuse time for Obsidian TNT.
     I:ObsidianTNTDelay=100
+
+    # Ops can bypass the block security restrictions if enabled.
     B:OpsBypassRestrictions=false
+
+    # Chance that osmium generates in a chunk. (0 to Disable)
     I:OsmiumPerChunk=0
+
+    # Max number of blocks in an osmium vein.
+    I:OsmiumVeinSize=8
+
+    # Delay in ticks before a player is teleported after clicking the Teleport button in the portable teleporter.
     I:PortableTeleporterDelay=0
+
+    # Add filled creative gas tanks to creative/JEI.
     B:PrefilledGasTanks=true
+
+    # If enabled makes Water and Heavy Water blocks be removed from the world on pump.
     B:PumpWaterSources=false
+
+    # Maximum buffer of an Entangoloporter frequency.
     D:QuantumEntangloporterEnergyTransfer=1.6E7
+
+    # Conversion multiplier from Joules to RF (Joules * RFToJoules = RF)
     D:RFToJoules=0.4
+
+    # Chance that salt generates in a chunk. (0 to Disable)
     I:SaltPerChunk=2
+
+    # Max number of blocks in a salt vein.
+    I:SaltVeinSize=6
+
+    # Chance of producing sawdust per operation in the precision sawmill when turning logs into planks.
+    D:SawdustChanceLog=1.0
+
+    # Chance of producing sawdust per operation in the precision sawmill when turning planks into sticks.
+    D:SawdustChancePlank=0.25
+
+    # Enable the spawning of baby skeletons. Think baby zombies but skeletons.
     B:SpawnBabySkeletons=false
+
+    # Amount of heat each Boiler heating element produces.
     D:SuperheatingHeatTransfer=10000.0
+
+    # Displayed temperature unit in Mekanism GUIs.
     S:"Temperature Units"=K
+
+    # Conversion multiplier from Joules to Tesla (Joules * TeslaToJoules = Tesla)
     D:TeslaToJoules=0.4
     D:ThermalEvaporationSpeed=1.0
+
+    # Chance that tin generates in a chunk. (0 to Disable)
     I:TinPerChunk=0
+
+    # Max number of blocks in a tin vein.
+    I:TinVeinSize=8
     B:UpdateNotifications=false
+
+    # Base factor for working out machine performance with upgrades - UpgradeModifier * (UpgradesInstalled/UpgradesPossible).
     I:UpgradeModifier=10
+
+    # TCP port for the Voice server to listen on.
     I:VoicePort=36123
     B:VoiceServerEnabled=false
```

Comment:
Replaced with WalkieTalkieServer (which should be set to false. Apparently we messed up the default there....)